server.js

```diff
@@ -20,7 +21,37 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to my Happy Thoughts API!')
+})
+
+app.get('/thoughts', async (req, res) => {
+  const tag = req.query.tag
+  const query = tag ? { tags: tag } : {}
```

Comment:
Oh this looks interesting! I see we had a different approach to this and now I'm super curious! What does this do? 😍

Reply:
This is filtering the data by categories/tags 🥇